config/gateway/default-gateway.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: networking.openfunction.io/v1alpha1
+kind: Gateway
+metadata:
+  name: openfunction
+  namespace: openfunction
+spec:
+  domain: ofn.io
+  clusterDomain: cluster.local
+  hostTemplate: "{{.Name}}.{{.Namespace}}.{{.Domain}}"
+  pathTemplate: "{{.Namespace}}/{{.Name}}"
+  httpRouteLabelKey: "app.kubernetes.io/managed-by"
+  gatewayRef:
+    name: contour
+    namespace: projectcontour
+  gatewaySpec:
+    listeners:
+      - name: ofn-http-external
```

Comment:
The default domain should have all the 3 listeners? 
- ofn-http-internal
- ofn-http-external
- ofn-https-external